src/service/components/input.js

```diff
@@ -276,8 +276,6 @@ class Controller {
      */
     _checkWayland() {
```

Comment:
This check was added in the initial disabling commit - should it be removed too, or is it still needed?

Reply:
Yeah, this whole function and all instances of `HAVE_WAYLAND` can go, actually.

Reply:
Should be done in 196166e